src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -156,6 +156,11 @@ var adyenHelperObj = {
   },
 
   getAdyenGivingConfig(order) {
```

Comment:
if `isAdyenGivingAvailable` is not used anywhere then it could be cleaned up

Reply:
It is being used on line 169